### ISSUE TITLE
Confusing version discrepancy

### ISSUE THREAD

**Issue by Yewzir** [arturo-lang/arturo#1660]
When building Arturo by yourself, the build version appears to be **0.9.84**.

```bash
oh@Yewzir:~/code-golf$ docker run -it --rm codegolf/lang-arturo
arturo v/0.9.84-alpha b/3190 (amd64/linux)
```

The project's home page, however, states its latest build version is **0.9.83**. That is very confusing, and I cannot help it.

What's up with that folks?

**Comment by RickBarretto** [comment]
Hi, and welcome to our community!

Well, this is actually a strange behavior. IDK if @drkameleon updated something recently, but my latest manual build is on 0.9.83 b/3112...

**Comment by Yewzir** [comment]
> Hi, and welcome to our community!
>
> Well, this is actually a strange behavior. IDK if @drkameleon updated something recently, but my latest manual build is on 0.9.83 b/3112... So, you should be using some build from the same version.

@RickBarretto 
Thank you very much.
Well, I'm really looking forward to hearing from him. Hopefully, the time's convenient for him anytime soon although there's no pressure. By any chance, would you happen to know a flag I can use to explicitly specify the build version? That should be a thing, shouldn't it?

**Comment by Yewzir** [comment]
I came across an interesting comment in `build.nims`, but it doesn't make sense really, from my point of view.

https://github.com/arturo-lang/arturo/blob/d1429540d88dccf0d3854e29cb2e8c40a8511497/build.nims#L163

**Comment by drkameleon** [comment]
@Yewzir Hi and welcome to our community! 😄 

The project's website (https://arturo-lang.io) reflects the latest official release, along with its documentation - that is: 0.9.83 (which was released about a year ago).

The master branch version is currently 0.9.84-alpha and will be released as 0.9.84 once everything is ready.

Re: build increments, that's basically something that I had done for myself and is to be used for tracking different builds when we build the project in "developer mode". But again, that's not really important, just for knowing where we are mostly!

That being said, I'm in the process of sanitizing the builds throughout the whole ecosystem, so minimal changes may be present... but again: mostly housekeeping! 😉

**Comment by drkameleon** [comment]
Regarding this: 

> By any chance, would you happen to know a flag I can temporarily use to explicitly specify the build version? That should be a thing, shouldn't it?

I totally agree.

As I said, I'm in the process of cleaning up the whole thing. And my goal when it's all finished is to actually support different build version when e.g. using Arturo via Docker *or* via a GitHub action, etc...

Just a little bit of patience! 😉 🚀

**Comment by Yewzir** [comment]
> @Yewzir Hi and welcome to our community! 😄

Thank you very much, sir.

> The project's website (https://arturo-lang.io) reflects the latest official release, along with its documentation - that is: 0.9.83 (which was released about a year ago).
> 
> The master branch version is currently 0.9.84-alpha and will be released as 0.9.84 once everything is ready.
> 
> Re: build increments, that's basically something that I had done for myself and is to be used for tracking different builds when we build the project in "developer mode". But again, that's not really important, just for knowing where we are mostly!
> 
> That being said, I'm in the process of sanitizing the builds throughout the whole ecosystem, so minimal changes may be present... but again: mostly housekeeping! 😉

You know, to be honest, I wrote that in a bit of a rush since I am in the process of implementing your project, initially as an experiment, on James Raspass' [Code Golf](https://code.golf). It all went so fast that I didn't even take a moment to consider what might be going on with that particular difference. As a matter of fact, I'm a regular user of Zig and regularly developing with their master build, which also happens to be one build _later_ than the latest stable release. What's even worse, is that such master builds probably live in a thousand places, so I sincerely apologize for my act of stupidity. Though thank you for the ~enlightenment~ reminder. For what it's worth, I really like your project and I wish you the best of luck in taking it to 1.0, and beyond.

Take care!

**Comment by drkameleon** [comment]
> > @Yewzir Hi and welcome to our community! 😄
> 
> Thank you very much, sir.

😄 

> You know, to be honest, I wrote that in a bit of a rush since I am in the process of implementing your project, initially as an experiment, on James Raspass' [Code Golf](https://code.golf). It all went so fast that I didn't even take a moment to consider what might be going on with that particular difference. As a matter of fact, I'm a regular user of Zig and regularly developing with their master build, which also happens to be one build _later_ than the latest stable release. What's even worse, is that such master builds probably live in a thousand places, so I sincerely apologize for my act of stupidity. Though thank you for the ~enlightenment~ reminder. For what it's worth, I really like your project and I wish you the best of luck in taking it to 1.0, and beyond.
> 
> Take care!

First of all, thanks A LOT for your kind words.

And... don't worry about anything. Feel free to shoot us with any question/doubt/issue you may come across (hint: for this type of things, I *think* Discord might even be better). That's exactly how the language may improve! 😉
